_includes/explore/_explore-resources.html

```diff
@@ -2493,13 +2493,13 @@
 
   .crds-legacy-styles .explore .map.multisitemap {
     background-color: #000 !important;
-    background: #000 url(https://crds-cms-uploads.imgix.net/content/images/mapmobilefade2.jpg?auto=compress,format) center center no-repeat;
+    background: #000 url(https://images.ctfassets.net/y3a9myzsdjan/5AvO7kqnypvJf4lUoO8bgG/2bd2bd6c35bd8e05f889a32367d42f47/crds-map-locations.jpg) center center no-repeat;
```

Comment:
We should change this to use imgix

Reply:
@silasbaur Can you please create a defect for these?